### PR TITLE
Fix wowfact

### PR DESCRIPTION
**Pull request by NadiaKishchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/models/streetcode/streetcode-types.model.ts [hunk 1]
```diff
@@ -158,5 +158,6 @@ export interface StreetcodeUpdate {
     toponyms: ToponymCreateUpdate[],
     images: ImageUpdate[],
     audios: AudioUpdate[],
+    imageDetailses: ImageDetails[],
```
Comment (Katerix) [inline src/models/streetcode/streetcode-types.model.ts:161]: spelling: imageDetailses => imageDetails

File: src/models/media/image.model.ts [hunk 2]
```diff
@@ -16,8 +16,9 @@ export interface ImageCreate {
 }
 export interface ImageDetails {
     id: number;
-    title: string;
-    alt: string;
+    title?: string;
+    alt?: string;
+    imageId:number;
```
Comment (Katerix) [inline src/models/media/image.model.ts:21]: add a space after : pls

**Review by MaksBrat — APPROVED** [review]

File: src/models/streetcode/streetcode-types.model.ts (hunk 1 above)
Comment (ormykhalyshyn) [reply src/models/streetcode/streetcode-types.model.ts:161]: +

**Review by Tysyatsky — APPROVED** [review]

File: src/models/streetcode/streetcode-types.model.ts (hunk 1 above)
Comment (ormykhalyshyn) [reply src/models/streetcode/streetcode-types.model.ts:161]: +

**Pull request merged**